.github/pull_request_template.md

```diff
@@ -0,0 +1,98 @@
+# SC-###
+
+<!--
+    If applicable, insert the Shortcut story number in the markdown header above.
+    The hyperlink will be filled in by GitHub magic ([autolink references](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources))
+--->
+
+## Proposed changes
+
+<!-- description and/or list of proposed changes -->
+
+---
+
+<!--
+    Please add/remove/edit any of the template below to fit the needs
+    of this specific PR
+--->
+
+## Reviewer notes
+
+<!--
+    Is there anything you would like reviewers to give additional scrutiny?
+--->
+
+## Setup
+
+<!--
+    Add any steps or code to run in this section to help others run your code:
+
+    ```sh
+    echo "Code goes here"
+    ```
+--->
+
+---
+
+## Code review steps
+
+### As the original developer, I have
+
+- [ ] Met the acceptance criteria, or will meet them in a subsequent PR
```

Comment:
I feel like these should be separate check boxes and should either include a link to the other PR or to the shortcut story.